pipelines/LsstCamImSim/ApPipeWithFakes.yaml

```diff
@@ -0,0 +1,34 @@
+description: AP pipeline with synthetic/fakes sources specialized for ImSim
+# (1) Execute `make_apdb.py`, e.g.,
+#     make_apdb.py -c isolation_level=READ_UNCOMMITTED
+#                  -c db_url="sqlite:////project/user/association.db"
+# (2) Run this pipeline, setting appropriate diaPipe configs
+#     (diaPipe configs should match the make_apdb.py configs)
+
+instrument: lsst.obs.lsst.LsstCamImSim
+imports:
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ApPipeWithFakes.yaml
+    exclude:  # These tasks come from LsstCamImSim/ApPipe.yaml instead
+      - processCcd
+  - location: $AP_PIPE_DIR/pipelines/LsstCamImSim/ApPipe.yaml
+    exclude:  # These tasks come from the ApPipeWithFakes.yaml
+      - retrieveTemplate
+      - subtractImages
+      - detectAndMeasure
+      - diaPipe
+      - transformDiaSrcCat
+
+tasks:
+  processVisitFakes:
+    class: lsst.pipe.tasks.processCcdWithFakes.ProcessCcdWithVariableFakesTask
+    config:
+      file: $AP_PIPE_DIR/config/HSC/processCcdWithFakes.py
+      connections.coaddName: parameters.CoaddName
+      insertFakes.doSubSelectSources: True
+      insertFakes.select_col: 'isVisitSource'
+      calibrate.photoCal.match.referenceSelection.magLimit.fluxField: i_flux
+      calibrate.photoCal.match.referenceSelection.magLimit.maximum: 22.0
```

Comment:
Remove `processVisitFakes` and its configs. This was needed for HSC, and not for other instruments (note the `HSC` in the file path).